src/bundles/files/actions.js

```diff
@@ -22,6 +22,27 @@ const realMfsPath = (path) => {
   return path
 }
 
+const stat = async (ipfs, hashOrPath) => {
+  const path = hashOrPath.startsWith('/')
+    ? hashOrPath
+    : `/ipfs/${hashOrPath}`
+
+  try {
+    const stats = await ipfs.files.stat(path)
+    return stats
+  } catch (e) {
+    if (e.toString().toLowerCase().includes('unixfs')) {
```

Comment:
This turned out to not be robust enough. 

I have old repo with various weird stuff in it and it breaks the pin screen due to error messages without `unixfs`:

![unhandled-errors-2019-08-20--16-32-16](https://user-images.githubusercontent.com/157609/63356895-d8091600-c368-11e9-8dcd-4e7848112820.png)

I proposed a robust fix in https://github.com/ipfs-shipyard/ipfs-webui/pull/1121